server/controllers/events/requests.js

```diff
@@ -0,0 +1,28 @@
+const slack = require('../../api/slack')
+const {
+  getAuthorizationRequestSlackBlocks,
+  getContributionRequestSlackBlocks
+} = require('../../helpers/requests')
+const { getBotAccessTokenForChannel } = require('../../helpers/tokens')
+
+const sendRequestEventToSlack = async (channelIds, blocks) => {
+  for (const channelId of channelIds) {
```

Comment:
style nit: I see other usages of `forEach` in this repo but not `for... of`.

Reply:
I personally prefer `for... of`. Is it better just to remain consistent in this case?

Reply:
I think consistency within a codebase is preferable.